tools/clang/test/DXC/Passes/ScalarReplHLSL/hitobject_make_scalarrepl.ll

```diff
@@ -0,0 +1,130 @@
+; RUN: %dxopt %s -hlsl-passes-resume -scalarrepl-param-hlsl -S | FileCheck %s
+; REQUIRES: dxil-1-9
```

Comment:
The IR tests don't run the validator and don't run with different versions of the compiler, so we don't need these REQUIRES lines.

Reply:
Removed the `REQUIRES` line